docs/articles/documentation/guides/advanced-guides/authentication.md

```diff
@@ -1,50 +1,37 @@
 ---
 layout: docs
-title: User Roles
-permalink: /documentation/test-api/authentication/user-roles.html
+title: Authentication
+permalink: /documentation/guides/advanced-guides/authentication.html
 ---
-# User Roles
+# Authentication
+
+TestCafe ships with the [user roles](#user-roles) mechanism that covers most authentication types that require user actions on the page.
+
+You can also test websites that use [HTTP Basic](https://en.wikipedia.org/wiki/Basic_access_authentication) and [NTLM](https://en.wikipedia.org/wiki/NT_LAN_Manager) authentication. See [HTTP/NTLM Authentication](#http-authentication) for more information on how to provide the credentials.
+
+## User Roles
 
 Many test scenarios involve the activity of more than one user. TestCafe addresses these scenarios by providing a convenient way
 to isolate authentication test actions and apply them easily whenever you need to switch the user account.
 
 A piece of logic that logs in a particular user is called a *role*. Define a role for each user account participating in your test.
```

Comment:
Are you explaining the meaning of a role here? If so, can we clarify this by saying:
A *role* defines permissions for users to perform specific actions. You can define a role for each user account in a test.

Reply:
A role is a piece of code, a function unique for each user. That's what I mean by saying `a piece of logic`. So, can I say:

*A role contains code that logs in a particular user. You can define a role for each user account in a test.*

Reply:
Okay, thank you